README.md

```diff
@@ -64,15 +64,26 @@ Also check out [@ProjectAntrea](https://twitter.com/ProjectAntrea) on Twitter!
 
 ## Features
 
-Antrea currently supports the following features:
-* IPv4 overlay network for a Kubernetes cluster. Geneve, VXLAN, GRE, or STT can
-be used as the encapsulation protocol.
-* [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies)
-implementation.
-* [Octant](https://github.com/vmware-tanzu/octant) UI plugin for monitoring
-Antrea components, which publish runtime information as
-[CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-* [IPsec encyption](/docs/ipsec-tunnel.md) of GRE tunnel traffic.
+* **Kubernetes-native**: Antrea follows best practices to extend the Kubernetes
+  APIs and provide familiar abstractions to users, while also leveraging
+  Kubernetes libraries in its own implementation.
+* **Run everywhere**: Multiple traffic modes (with or without overlay) are
+  supported to ensure that Antrea can run in private clouds, public clouds and
+  at the edge.
+* **Windows Node support**: Thanks to the portability of Open vSwitch, Antrea
+  can use the same data plane implementation on both Linux and Windows
+  Kubernetes Nodes.
+* **Comprehensive policy model**: Antrea provides a comprehensive network policy
+  model, which builds upon Kubernetes Network Policies with new features such as
+  policy tiering, rule priorities and cluster-level policies.
+* **Troubleshooting and monitoring tools**: Antrea comes with a CLI tool
+  (antctl) and an Octant UI plugin, it exposes Prometheus metrics and supports
+  exporting network flow information which can be visualized in Kibana
+  dashboards.
+* **Encryption**: Encryption of inter-Node Pod traffic with IPsec tunnels when
+  using an overlay Pod network.
+* **Easy deployment**: Antrea is deployed by applying a single YAML manifest
+  file.
 
```

Comment:
Still good to add the changelog doc link here?

Reply:
We can do that, is this something we discussed before?

Reply:
Yes, I think we talked about that when discussing the features section changes.

Reply:
I added the link to CHANGLEOG.md, but I think @McCodeman  is working on a detailed table with a comprehensive list of features, their description, the version in which they are introduced. When this is ready, we should replace this link with a link to the table.